python-main.js

```diff
@@ -627,11 +627,22 @@ function web_editor(config) {
         $("#command-share").click(function () {
             doShare();
         });
+
+        function formatHelpPanel(){
+            if($(".helpsupport_container").offset().left !== $("#command-help").offset().left && $(window).width() > 620){
+                console.log("wow");
```

Comment:
I assume you forgot to remove this one? :)

Reply:
yeah forgot to remove it, seeing if the if statement was working at the correct point, will remove it now